### ISSUE TITLE
Add migration to rebuild cached nomenclature date

### ISSUE THREAD

**Issue by mjy** [SpeciesFileGroup/taxonworks#2687]
Once #2686 is resolved we should include a migration to update the field so we can properly depend on the values there.

**Comment by mjy** [comment]
#2686 is resolved.  Wondering if this should be a task, or a migration.

**Comment by proceps** [comment]
it should be fixed with soft_validation fixes. We need a task to run soft_validations and fixes on a project.

**Comment by mjy** [comment]
> it should be fixed with soft_validation fixes. We need a task to run soft_validations and fixes on a project.

Yes, but also in theory we should never need a soft validation fix in the long run.  This proposal here is to get everything on the same page in the most efficient way possible.

**Comment by LocoDelAssembly** [comment]
If you add a migration for this please make sure it won't take too long to execute otherwise will cause deploy to fail.

**Comment by mjy** [comment]
I think I'll just make a rake task for now.

**Comment by proceps** [comment]
If you do it, do not forget also set cached_is_valid.

**Comment by mjy** [comment]
`cached_is_valid` is already correct via migration

**Comment by proceps** [comment]
There is number of issues getting accumulated which could be resolved with soft validation dashboard #2546
